config/environment.default.js

```diff
@@ -9,11 +9,11 @@ module.exports = {
   },
   // The REST API server settings.
   rest: {
-    ssl: true,
+      ssl: true,
     host: 'dspace7.4science.cloud',
-    port: 443,
+      port: 443,
     // NOTE: Space is capitalized because 'namespace' is a reserved string in TypeScript
-    nameSpace: '/server/api'
+      nameSpace: '/server/api'
```

Comment:
Answering a few of your issues, the rest will be covered by @artlowel:

2. Fixed this issue, it might have been introduced by a merge conflict
3. This issue was already fixed in [a later PR](https://github.com/DSpace/dspace-angular/pull/541), where we changed a significant part of the way the values are rendered. 
4. Same as 3.

Reply:
Reverted indenting